docs/pods.md

```diff
@@ -55,3 +55,12 @@ _Why not just run multiple programs in a single (Docker) container?_
 _Why not support affinity-based co-scheduling of containers?_
 
 That approach would provide co-location, but would not provide most of the benefits of pods, such as resource sharing, IPC, guaranteed fate sharing, and simplified management.
+
+## Durability of pods (or lack thereof)
+
+Pods should never be treated as durable pets. They won't survive scheduling failures, node failures, or other evictions, such as due to lack of resources or for node maintenance. In general, users shouldn't create pods directly. They should almost always use controllers (e.g., replication controller), even for singletons.
```

Comment:
can we change "shouldn't" to "shouldn't need to"

It changes it from an order to our users, to a directive for us.  Instead of "don't do this" we should say: "we made it so easy to to the other thing, you want to do that"